tensorflow/python/kernel_tests/nn_ops/BUILD

```diff
@@ -28,6 +28,27 @@ test_suite(
             [basename + "_gpu" for basename in CONV_TEST_BASENAMES],
 )
 
+# tensorflow/python/kernel_tests/nn_ops/BUILD
+cuda_py_strict_test(
+    name = "separable_conv2d_patch_test",
+    size = "small",
+    srcs = ["separable_conv2d_patch_test.py"],
+    deps = [
+        "//tensorflow/python/framework:constant_op",
+        "//tensorflow/python/framework:errors",
+        "//tensorflow/python/framework:for_generated_wrappers",
+        "//tensorflow/python/framework:test_lib",
+        "//tensorflow/python/ops:array_ops",
+        "//tensorflow/python/ops:nn_impl",
+        "//tensorflow/python/ops:nn_ops",
+        "//tensorflow/python/ops:gradient_checker_v2",
+        "//tensorflow/python/platform:client_testlib",
+        "//third_party/py/numpy",
+        "@absl_py//absl/testing:parameterized",
+    ],
+    tags = ["no_pip", "no_gpu"],
```

Comment:
Why did you add these tags? They disable tests and should only be added if we know the test needs to be disabled